app/scripts/modules/oracle/instance/details/instance.details.controller.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+import {INSTANCE_READ_SERVICE} from 'core/instance/instance.read.service';
+import {INSTANCE_WRITE_SERVICE} from 'core/instance/instance.write.service';
+import {CLOUD_PROVIDER_REGISTRY} from 'core/cloudProvider/cloudProvider.registry';
+
+let angular = require('angular');
```

Comment:
@anotherchrisberry sure that change looks great! 👍  I'll wait until your changes are merged and then update this.